From bc1407a89d3a9e5da8ea7a0f9de440f1ee9e8ddd Mon Sep 17 00:00:00 2001
From: Adrian DC <radian.dc@gmail.com>
Date: Fri, 9 Mar 2018 06:56:34 +0100
Subject: [PATCH 60/74] libfm_jni: Remove unused variables

 * libfm_jni/FmPerformanceParams.cpp:
    443:16: error: unused variable 'sinr'

 * libfm_jni/ConfigFmThs.cpp:
    425:17: error: unused variable 'ret'
    433:9: error: unused variable 'value'
    566:9: error: unused variable 'index'

 * libfm_jni/ConfFileParser.cpp:
    328:16: error: unused variable 'num_of_keys'
    536:9: error: unused variable 'grp_name'
    537:16: error: unused variable 'len'
    732:13: error: unused function 'key_exist'

 * libfm_jni/FmRadioController.cpp:
    651:9: error: unused variable 'stationList

 * libfm_jni/FmPerformanceParams.cpp:
    442:16: error: unused variable 'sinr'

Change-Id: I9c4839d1bc8b64c536b8a7db7c2094001f2562ba
---
 libfm_jni/ConfFileParser.cpp      | 50 -------------------------------
 libfm_jni/ConfigFmThs.cpp         |  3 --
 libfm_jni/FmPerformanceParams.cpp |  1 -
 libfm_jni/FmRadioController.cpp   |  1 -
 4 files changed, 55 deletions(-)

diff --git a/libfm_jni/ConfFileParser.cpp b/libfm_jni/ConfFileParser.cpp
index 00146f6..5897992 100644
--- a/libfm_jni/ConfFileParser.cpp
+++ b/libfm_jni/ConfFileParser.cpp
@@ -325,7 +325,6 @@ char **get_keys
 {
   unsigned int grp_hash_code;
   unsigned int grp_index;
-  unsigned int num_of_keys;
   unsigned int i;
   unsigned int j = 0;
   unsigned int key_len;
@@ -533,8 +532,6 @@ static char parse_line
 )
 {
   const char *line_begin;
-  char *grp_name;
-  unsigned int len;
 
   if((line == NULL) || (key_file == NULL)) {
       ALOGE("key file or line is null\n");
@@ -732,53 +729,6 @@ static char line_is_grp
   }
 }
 
-static char key_exist
-(
-  const group_table *key_file,
-  const char *cur_grp,
-  const char *key
-)
-{
-  unsigned int grp_hash_code;
-  unsigned int key_hash_code;
-  unsigned int grp_index;
-  unsigned int key_index;
-  group *grp = NULL;
-  key_value_pair_list *list = NULL;
-
-  if((key_file != NULL) && (cur_grp != NULL)
-      && (key != NULL) && ((key_file->grps_hash != NULL))
-      && (strcmp(key, ""))) {
-     grp_hash_code = get_hash_code(cur_grp);
-     grp_index = (grp_hash_code % key_file->grps_hash_size);
-     grp = key_file->grps_hash[grp_index];
-     key_hash_code = get_hash_code(key);
-     while((grp != NULL)) {
-           if(!strcmp(cur_grp, grp->grp_name)) {
-              key_index = (key_hash_code % grp->keys_hash_size);
-              if(grp->list)
-                 list = grp->list[key_index];
-              while((list != NULL) && strcmp(key, list->key)) {
-                    list = list->next;
-              }
-              if(list != NULL){
-                  return TRUE;
-              }else{
-                  return FALSE;
-              }
-           }
-           grp = grp->grp_next;
-     }
-     if(!grp) {
-        return TRUE;
-     }else {
-        return FALSE;
-     }
-  }else {
-     return FALSE;
-  }
-}
-
 //checks validity of key
 //a valid key must start in
 //a seperate line and key must
diff --git a/libfm_jni/ConfigFmThs.cpp b/libfm_jni/ConfigFmThs.cpp
index c04ef8a..369a8b4 100644
--- a/libfm_jni/ConfigFmThs.cpp
+++ b/libfm_jni/ConfigFmThs.cpp
@@ -422,7 +422,6 @@ void ConfigFmThs :: set_hybrd_list
     UINT fd
 )
 {
-    signed char ret = FM_SUCCESS;
     char **keys = NULL;
     char **keys_cpy = NULL;
     char *key_value = NULL;
@@ -430,7 +429,6 @@ void ConfigFmThs :: set_hybrd_list
     unsigned int *freqs_array = NULL;
     signed char *sinrs_array = NULL;
     char *sinrs = NULL;
-    int value;
     unsigned int freq_cnt = 0;
     unsigned int sinr_cnt = 0;
     FmPerformanceParams perf_params;
@@ -563,7 +561,6 @@ void  ConfigFmThs :: SetRxSearchAfThs
     const char *file, UINT fd
 )
 {
-    int index;
     struct NAME_MAP *found;
     char **grps = NULL;
     char **grps_cpy = NULL;
diff --git a/libfm_jni/FmPerformanceParams.cpp b/libfm_jni/FmPerformanceParams.cpp
index c5cfe68..828f63c 100644
--- a/libfm_jni/FmPerformanceParams.cpp
+++ b/libfm_jni/FmPerformanceParams.cpp
@@ -440,7 +440,6 @@ signed char FmPerformanceParams :: SetHybridSrchList
    struct v4l2_ext_control ext_ctl;
    struct v4l2_ext_controls v4l2_ctls;
    unsigned int freq;
-   signed char sinr;
    unsigned int size = 0;
    char *data = NULL;
    signed char ret = FM_FAILURE;
diff --git a/libfm_jni/FmRadioController.cpp b/libfm_jni/FmRadioController.cpp
index ca524e7..a002121 100644
--- a/libfm_jni/FmRadioController.cpp
+++ b/libfm_jni/FmRadioController.cpp
@@ -648,7 +648,6 @@ int FmRadioController :: GetStationList
     int ret;
     ULINT lowBand, highBand;
     int station_num = 0;
-    int stationList[FM_RX_SRCHLIST_MAX_STATIONS];
     int tmpFreqByte1=0;
     int tmpFreqByte2=0;
     int freq = 0;
-- 
2.17.1

